### ISSUE TITLE
 nevim Not enough arguments for function

### ISSUE THREAD

**Issue by peterwillcn** [jodosha/vim-godebug#17]
Error detected while processing function godebug#debug:
line    2:
E119: Not enough arguments for function: go#term#new
E15: Invalid expression: go#term#new(a:bang, ["dlv", "debug", "--init=" . g:godebug_breakpoints_file])

**Comment by iloginow** [comment]
Same issue here.

**Comment by dude333** [comment]
Apparently, vim-go change the parameters for `go#term#new`. 

This plugin calls it as [this](https://github.com/jodosha/vim-godebug/blob/4ad62ca7612666890b08b7906e16b8c3c5c582f4/plugin/godebug.vim#L67):
```
go#term#new(a:bang, ["dlv", "debug", "--init=" . g:godebug_breakpoints_file])
```
But look at this function on vim-go [code](https://github.com/fatih/vim-go/blob/32026717bd548db8274ba6ca309b648cd779396d/autoload/go/term.vim#L7):

```
function! go#term#new(bang, cmd, errorformat) abort
  return go#term#newmode(a:bang, a:cmd, a:errorformat, go#config#TermMode())
endfunction
```

I fixed it by add this parm on godebug.vim:
```
function! godebug#debug(bang, ...) abort
  call godebug#writeBreakpointsFile()
  return go#term#new(a:bang, ["dlv", "debug", "--init=" . g:godebug_breakpoints_file], "%-G#\ %.%#")
endfunction

function! godebug#debugtest(bang, ...) abort
  call godebug#writeBreakpointsFile()
  return go#term#new(a:bang, ["dlv", "test", "--init=" . g:godebug_breakpoints_file], "%-G#\ %.%#")
endfunction
```

Need to check better this errorformat parm though (http://vimdoc.sourceforge.net/htmldoc/quickfix.html#errorformat).

**Comment by jodosha** [comment]
Fixed by #18
